sympy/core/expr.py

```diff
@@ -2645,7 +2645,37 @@ def series(self, x=None, x0=0, n=6, dir="+", logx=None):
         If ``x=None`` and ``self`` is univariate, the univariate symbol will
         be supplied, otherwise an error will be raised.
 
-        >>> from sympy import cos, exp
+        Parameters
+        ==========
+
+        expr : Expression
+               The expression whose series is to be expanded.
```

Comment:
I'd expect it to be indented 4*n characters, and some below
```suggestion
            The expression whose series is to be expanded.
```

Reply:
Its okay. I checked the numpydoc format now.